frontend/model/database.js

```diff
@@ -26,26 +26,35 @@ const localforage = {
               reject(new Error('Unsupported characters in name: -'))
               return
             }
-            const request = self.indexedDB.open(name + '--' + storeName)
 
-            // Create the object store if it doesn't exist
-            request.onupgradeneeded = (event) => {
-              const db = event.target.result
-              db.createObjectStore(storeName)
-            }
+            const openDB = (version?: number) => {
+              const request = self.indexedDB.open(name + '--' + storeName, version)
```

Comment:
Could you please add a comment here explaining:

1. What this second parameter to `.open` is
2. Why this is needed
3. A reminder to users that if they get the `'DB is blocked'` error, to try closing all open tabs except one (and why that is necessary)